debian/changelog

```diff
@@ -1,3 +1,8 @@
+singularity-container (2.4.6-1) unstable; urgency=high
```

Comment:
I think this should actually be 2.4.6rc2-1 as well just so that we differentiate between an actual release here.  We can change it when we do the real release.  Thoughts?

Reply:
Debian appears to be the same:

https://www.debian.org/doc/debian-policy/#version-numbers-based-on-dates